.circleci/config.yml

```diff
@@ -48,14 +41,20 @@ commands:
             sudo apt-get update
             sudo apt-get install -y cmake curl clang llvm
             rustup component add clippy rustfmt
+            rustup toolchain install $RUST_NIGHTLY
+      - run:
+          name: Set cargo Environment
+          command: |
+            echo 'export CARGO=$(rustup which --toolchain $RUST_NIGHTLY cargo)' >> $BASH_ENV
+            echo 'export CARGOFLAGS=-Zfeatures=all' >> $BASH_ENV
```

Comment:
Is this the flag that turns on the nightly feature unification separation between dev/non-dev dependencies?   Would you mind adding a comment about what behavior it changes in cargo?

Reply:
There are three features settings plus "all". itarget, dev-dep, and build-dep. Documentation is here: https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#features. I'll add a link to this in the comments.